core/services/fluxmonitorv2/submission_checker.go

```diff
@@ -13,10 +13,10 @@ type SubmissionChecker struct {
 }
 
 // NewSubmissionChecker initializes a new SubmissionChecker
-func NewSubmissionChecker(min *big.Int, max *big.Int, precision int32) *SubmissionChecker {
+func NewSubmissionChecker(min *big.Int, max *big.Int) *SubmissionChecker {
 	return &SubmissionChecker{
-		Min: decimal.NewFromBigInt(min, -precision),
-		Max: decimal.NewFromBigInt(max, -precision),
+		Min: decimal.NewFromBigInt(min, 0),
```

Comment:
This is fix for the immediate bug - now we just do the comparison post pipeline run all with scaled values